model.py

```diff
@@ -0,0 +1,250 @@
+"""This file defines a tranformer-based model to detect intents.
+
+This file implements a transformer-based model to detect intents and the corresponding 
+slots. The slot corresponding to a token is predicted by passing the contextual embeddings
```

Comment:
Break lines at 80 chars:
http://google.github.io/styleguide/pyguide.html#32-line-length

You can try using https://github.com/google/yapf for formatting your code.

Reply:
You can consider add flake8 check: https://stackoverflow.com/questions/42325453/per-project-flake8-max-line-length